plugins/release-manager/src/components/AndroidReleaseNotesPane.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { ChangeEvent, FC } from 'react';
+import { AndroidReleaseNote } from '../types';
+import { RmAccordion } from './RmAccordion';
+import { TextField, Typography } from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+import { Spacer } from './Spacer';
+import { StatusContainer } from './StatusContainer';
+import { InfoIcon } from './InfoIcon';
+import { StatusItem } from './StatusItem';
+import { getAndroidReleaseNotesStatus } from '../utils/status';
+
+type Props = {
+  releaseNotes: AndroidReleaseNote[];
+};
+
+export const AndroidReleaseNotesPane: FC<Props> = ({ releaseNotes }) => {
```

Comment:
We avoid using `FC`, see https://backstage.io/docs/architecture-decisions/adrs-adr006